<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IGLMeshManager</class>
 <widget class="QFrame" name="IGLMeshManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QFrame" name="frame1">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>8</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QPushButton" name="loadIGLMeshButton">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Load IGLMesh</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="cleanIGLMeshButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Clean IGLMesh</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="saveIGLMeshButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Save IGLMesh</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>8</y>
     <width>131</width>
     <height>124</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="3" column="0">
     <widget class="QCheckBox" name="wireframeIGLMeshCheckBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Wireframe</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QRadioButton" name="smoothIGLMeshRadioButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Smooth</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QRadioButton" name="pointsIGLMeshRadioButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Points</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="flatIGLMeshRadioButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Flat</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
